docs/revamp-2026.md

```
@@ -0,0 +1,97 @@
+- **Geist Mono** (monospace): Code blocks in writings
+
+### Animations
+- **Motion** (formerly Framer Motion) for all animations
+- Declarative API for complex sequences
+
+## Deployment & SEO
+- **Cloudflare Pages** (already configured)
+- Sitemap, RSS, LLMs.txt, Open Graph (already configured via astro.config.mjs)
+
+## File Structure
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)